projects/kit/directives/highlight/highlight.directive.ts

```diff
@@ -20,6 +22,7 @@ export class TuiHighlight implements OnChanges {
     private readonly el = tuiInjectElement();
     private readonly renderer = inject(Renderer2);
     private readonly doc = inject(DOCUMENT);
+    private readonly options = inject(TUI_HIGHLIGHT_OPTIONS);
```

Comment:
We do not need to store it on the class, can just write `public tuiHighlightColor = inject(TUI_HIGHLIGHT_OPTIONS).highlightColor` directly until we get any other option.

Reply:
fix Sorry, my fault